src/sd-jwt/errors.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * When claims are requested but not found in the credential
+ *
+ */
+export class ClaimsNotFoundBetweenDisclosures extends Error {
+  code = "ERR_CLAIMS_NOT_FOUND";
+
+  /** The Claims not found */
+  claims: string[];
+
+  constructor(claims: string | string[]) {
+    const c = Array.isArray(claims) ? claims : [claims];
+    const message = `Some requested claims are not present in the disclosurable values, claims: ${c.join(
+      ", "
+    )}`;
+    super(message);
+    this.claims = c;
```

Comment:
```suggestion
      const claimsArray = Array.isArray(claims) ? claims : [claims];
      super(`Some requested claims are not present in the disclosurable values: ${claimsArray.join(", ")}`);
      this.claims = claimsArray;
```

Reply:
Addressed in 3efbcee36cac790849c39a455cac6c45ea8ce5e6.